.github/workflows/main-branches.yml

```
@@ -11,6 +11,26 @@ env:
   EXECUTABLES: "raytracer"
 
 jobs:
+  compare-keys:
+    name: Compare CLI 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)